draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -549,7 +549,7 @@ If the Issuer wants to enable Key Binding, it includes a public key
 associated with the Holder, or a reference thereto.
 
 It is out of the scope of this document to describe how the Holder key pair is
-established. For example, the Holder MAY provide a key pair to the Issuer,
+established. For example, the Holder MAY create a key pair and provide a public key to the Issuer,
```

Comment:
feedback received: 
> "It is out of the scope of this document to describe how the Holder key pair is established. For example, the Holder MAY provide a key pair to the Issuer,…" Requiring the holder to provide also the private key to the issuer appears not to be a good example.

suggestion was "the Holder MAY provide a public key to the Issuer and prove possession of the corresponding private key" but I slightly played with the words.